app/assets/javascripts/discourse/app/components/topic-timer-info.js

```diff
@@ -67,7 +67,9 @@ export default Component.extend({
 
       let options = {
         timeLeft: duration.humanize(true),
-        duration: moment.duration(durationMinutes, "minutes").humanize(),
+        duration: moment
+          .duration(durationMinutes, "minutes")
+          .humanize({ s: 60, m: 60, h: 24 }),
```

Comment:
This makes the message in the UI more accurate, so 47 minutes doesn't say "in 1 hour" etc.